README.md

```diff
@@ -94,14 +94,16 @@ export default MyNode;
     - [DEPRECATED: Hard Sequence Handlers](#deprecated-hard-sequence-handlers)
 - [Interaction with React](#interaction-with-react)
 - [HotKeys components](#hotkeys-components)
-    - [How action handlers are resolved](#how-action-handlers-are-resolved)
 - [HotKeys component API](#hotkeys-component-api)
 - [withHotKeys HoC API](#withhotkeys-hoc-api)
     - [Simple use-case](#simple-use-case)
     - [Pre-defining default prop values](#pre-defining-default-prop-values)
 - [GlobalHotKeys component](#globalhotkeys-component)
-    - [How actions and handlers are resolved](#how-actions-and-handlers-are-resolved)
 - [GlobalHotKeys component API](#globalhotkeys-component-api)
+- [How actions are resolved](#how-actions-are-resolved)
```

Comment:
🐥 Move from individual sections on how action handlers are resolved for `<HotKeys />` and `<GlobalHotkeys />`, to a single section that describes how `react-hotkeys` resolves actions as a whole.